group_vars/rhcs.yml.sample

```diff
@@ -29,7 +29,7 @@ dummy:
 #  mimic: 13
 
 # Directory to fetch cluster fsid, keys etc...
-fetch_directory: ~/ceph-ansible-keys
+#fetch_directory: fetch/
```

Comment:
why did you change that?

Reply:
It was generated when run ./generate_group_vars_sample.sh

Reply:
@mgornikov probably due to a bad rebase on master